package.json

```diff
@@ -38,9 +38,34 @@
       "npm run server:lint"
     ]
   },
+  "eslintConfig": {
+    "plugins": [
+      "prettier"
+    ],
+    "extends": [
+      "plugin:prettier/recommended"
+    ],
+    "rules": {
+      "no-throw-literal": "off",
+      "no-undef": "off",
+      "prettier/prettier": "error"
+    }
+  },
+  "prettier": {
+    "printWidth": 100,
+    "singleQuote": true,
+    "trailingComma": "all",
+    "endOfLine": "lf"
```

Comment:
It seems to be the default value, but we can leave it in, it's not that important. Or could it make sense in some case?

Reply:
nope, i thought with endOfLine settings set to lf explicitly will force vscode to change it's EOL sequence by default, maybe i'll try to create an editor conf file to see if its work